server/authz/consts.go

```diff
@@ -20,6 +20,10 @@ const (
 	// PolicyMapPath is the path to the policy map.
 	policyMapPath = "/etc/numaflow/rbac-policy.csv"
 
+	// RbacPropertiesPath is the path to the rbac properties file. It includes configuraion for authorization like
+	// scope, default policy etc.
+	RbacPropertiesPath = "/etc/numaflow/rbacConf.yaml"
```

Comment:
should it be exported?

Reply:
Not using it outside scope after change, removing